packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -110,13 +110,13 @@ export function RepositoryMixin<T extends Class<any>>(superClass: T) {
     ): Binding<D> {
       // We have an instance of
       if (dataSource instanceof juggler.DataSource) {
-        const key = `datasources.${name || dataSource.name}`;
+        const key = `datasources.${name ?? dataSource.name}`;
         return this.bind(key)
           .to(dataSource)
           .tag('datasource');
       } else if (typeof dataSource === 'function') {
         const binding = createBindingFromClass(dataSource, {
-          name: name || dataSource.dataSourceName,
+          name: name ?? dataSource.dataSourceName,
```

Comment:
Ditto.

Before this change, an empty name `''` will be converted to `dataSource.name`. With the change in place, we will preserve the empty string. I think that's not correct. Thoughts?

Reply:
AFAICT, this is not resolved. Empty name `''` will be passed to `createBindingFromClass`. Does that helper correctly handle empty string in `name` option? Shouldn't we use `dataSource.dataSourceName` instead?